cpp/map_module/algorithm/map.cpp

```diff
@@ -0,0 +1,34 @@
+#include <list>
+#include <sstream>
+
+#include "map.hpp"
+
+const auto number_of_elements_in_pair = 2;
+
+void Map::FromPairs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto list = arguments[0].ValueList();
+
+    mgp::Map pairs_map;
+
+    for (const auto inside_list : list) {
+      if (inside_list.ValueList().Size() != number_of_elements_in_pair) {
+        throw mgp::IndexException("Number of elements in a pair is not right.");
```

Comment:
Did you try to use the `fmt::format` library here?

Reply:
It would be helpfult to output 2 as number

Reply:
sorry I wasn't sure how to work with it... when I install that library locally do I somehow have to include it in this PR?

Reply:
# Add all module files related to graph util module
set(collections_module_src
    collections_module.cpp ...)

add_query_module(collections 1 "${collections_module_src}")

target_link_libraries(collections_module_src PRIVATE fmt::fmt) <- try with this